deployment/terraform/cdn.tf

```diff
@@ -61,6 +61,28 @@ resource "aws_cloudfront_distribution" "cdn" {
     max_ttl                = 300
   }
 
+  ordered_cache_behavior {
+    path_pattern     = "tile/*"
+    allowed_methods  = ["GET", "HEAD", "OPTIONS"]
+    cached_methods   = ["GET", "HEAD", "OPTIONS"]
+    target_origin_id = "originAlb"
+
+    forwarded_values {
+      query_string = true
+      headers      = ["Referer"]
```

Comment:
It is unclear why `Referer` is being kept in this list. If because we want to distinguish between staging and production, each will have its own CloudFront distribution.

Reply:
The reasoning is that if someone is hotlinking to this layer from another site, the Referer field will be `data-pirates.xyz`, which would not get the cached tile, but go straight to origin, which will 401 the request. Otherwise, they would get the cached tiles, so the hotlinking would partially work (only for the already cached tiles, won't be able to request new ones).

Reply:
Ah, OK. 👍

Reply:
I will also add a comment to that effect so it's clearer going forward